.gitignore

```diff
@@ -28,3 +28,7 @@ Session.vim
 *~
 tags
 .pem
+
+
+# Post-test cruft
+Gemfile.lock
```

Comment:
See https://github.com/BonnyCI/bonnyci.org/pull/43 for the solution to that problem. :)